test/table.js

```diff
@@ -1854,6 +1879,18 @@ describe('Bigtable/Table', function() {
         });
       });
     });
+
+    it('should use an appProfileId', function(done) {
```

Comment:
This test is misplaced as well. Instead, it should live under `sampleRowKeysStream`, as that is the function that was modified. The new test can go right after:

```js
  describe('sampleRowKeysStream', function() {
    it('should provide the proper request options', function(done) {
      // ...
    });

    // here
```